chgnet/graph/converter.py

```diff
@@ -52,17 +59,11 @@ def __init__(
         )
 
         # Set graph conversion algorithm
-        if algorithm == "fast":
-            try:
-                from chgnet.graph.cygraph import make_graph
-
-                self._make_graph = make_graph
-                self.create_graph = self._create_graph_fast
-                self.algorithm = "fast"
-            except (ImportError, AttributeError):
-                self.create_graph = self._create_graph_legacy
-                self.algorithm = "legacy"
-        elif algorithm == "legacy":
+        self._make_graph = None
+        if algorithm == "fast" and make_graph is not None:
+            self.create_graph = self._create_graph_fast
```

Comment:
We should probably print a warning or even error when `algorithm == "fast" and make_graph is None`.

Reply:
Probably for the best. I added warnings when fast is unavailable or and unrecognised argument is passed